File: ArticuloApp.Library/src/STL/Notificacion.java
package STL;


import STL.Enumeraciones;

/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 *
 * @author Administrator
 */
public class Notificacion {
    private Cliente remitente;
    private Cliente destinatario;
    private String mensaje;
    private Enumeraciones.EstadosNotificacion estado;

    public Cliente getRemitente() {
        return remitente;
    }

    public void setRemitente(Cliente remitente) {
        this.remitente = remitente;
    }

    public Cliente getDestinatario() {
        return destinatario;
    }

    public void setDestinatario(Cliente destinatario) {
        this.destinatario = destinatario;
    }

    public String getMensaje() {
        return mensaje;
    }

    public void setMensaje(String mensaje) {
        this.mensaje = mensaje;
    }

    public Enumeraciones.EstadosNotificacion getEstado() {
        return estado;
    }

    public void setEstado(Enumeraciones.EstadosNotificacion estado) {
        this.estado = estado;
    }
}

File: ArticuloApp/ArticuloApp.Entidades/Notificacion.cs
namespace ArticuloApp.Entidades
{
    public class Notificacion
    {
        public  int id { get; set; }
        public Cliente remitente { get; set; }
        public Cliente destinatario { get; set; }
        public string asunto { get; set; }
        public string mensaje { get; set; }
        public Enumeraciones.EstadosNotificacion estado { get; set; }

        public Notificacion(Cliente remitente, Cliente destinatario, string asunto, string mensaje, Enumeraciones.EstadosNotificacion estado)
        {
            this.id = default(int);
            this.remitente = remitente;
            this.destinatario = destinatario;
            this.asunto = asunto;
            this.mensaje = mensaje;
            this.estado = estado;
        }

        public Notificacion()
        {
            this.id = default(int);
            this.remitente = new Cliente();
            this.destinatario = new Cliente();
            this.asunto = string.Empty;
            this.mensaje = string.Empty;
            this.estado = Enumeraciones.EstadosNotificacion.PorEnviar;
        }
    }
}
